docs/source/conf.py

```diff
@@ -37,6 +37,10 @@
               'sphinx.ext.intersphinx', 'sphinx.ext.coverage',
               'sphinx.ext.autosummary', 'sphinx.ext.doctest']
 
+nitpick_ignore = [('py:obj', 'buttord'), ('py:obj', 'buttap'), ('py:obj', 'sosfiltfilt'), 
```

Comment:
interesting solution. can you put a comment above this line mentioning it is to ignore the scipy imports that are happening automatically through autodoc. (it is actually because of the template I wrote for sphinx which just keeps pulling everything, so eventually I need to fix that)